### DISCUSSION TITLE
Growth!

### DISCUSSION THREAD

**Discussion by hazah** [hazah/polyverse#15] [Announcements]
Welcome @papersheepdog and @eonhetwo into this long coming, overly elaborate, but fundamentally simple project that's been baking in the ether for a decade. I couldn't be in better company in seeing this as through as we can do ;)

**Comment by papersheepdog** [comment]
You have my axe

**Comment by hazah** [comment]
@papersheepdog @eonhetwo More practically, I think this is the platform to discuss an issue before it becomes an `issue`.  This way we can get all eyes and figure out what the demand might actually be before it's declared.

**Comment by hazah** [comment]
Specifically, there's a link to `Create issue from discussion` off to the side menu right in the discussion. Ie we have another protocol we can use to tame the beast!

**Reply by psidekick** [reply]
Helloooo

**Comment by eonhetwo** [comment]
I'll lend my bow to your will, sir
